erpnext/accounts/doctype/payment_request/test_payment_request.py

```diff
@@ -178,9 +178,9 @@ def test_payment_channels(self):
 		pr.reload()
 
 		self.assertEqual(pr.payment_channel, "Email")
-		self.assertEqual(pr.mute_email, False)
+		self.assertEqual(pr.mute_email, True)
 
-		self.assertIsNone(pr.payment_url)
+		self.assertEqual(pr.payment_url, PAYMENT_URL)
```

Comment:
I beleive this should be part of https://github.com/frappe/erpnext/pull/40845/commits/bc1a7c69a8188e5af2d67bee1cfd7817742bb85a

Waiting for test runs

Reply:
Effectively so it was. Removed.